pkg/cmd/admin/registry/registry.go

```diff
@@ -251,16 +263,8 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 						SecurityContext: &kapi.SecurityContext{
 							Privileged: &mountHost,
 						},
-						LivenessProbe: &kapi.Probe{
-							InitialDelaySeconds: 3,
-							TimeoutSeconds:      5,
```

Comment:
Those are the values used with router probes now. It makes sense to increase initial delay because of additional backend drivers supported (swift, azure, gsc). Some of them do authentication during their initialization which prolongs registry's startup. On the other hand, `TimeoutSeconds` defaults to 1 which may be too short interval indeed, I'll revert back to the 5s.